### PR TITLE
chore(suite): jest.config cleanup

### PR DESCRIPTION
**Pull request by mroz22**
take it or close it. I don't really care. I am just periodically getting triggered by `collectCoverageFrom:` in jest settings. Why not collect from everywhere? This way, some tests we actually have are not even included in coverage.

### CODE REVIEW

File: packages/suite/jest.config.js [hunk 1]
```diff
@@ -31,37 +31,20 @@ module.exports = {
...
+            statements: 33,
+            branches: 22,
+            functions: 34,
+            lines: 22,
         },
     },
-    modulePathIgnorePatterns: [
-        'node_modules',
-        '<rootDir>/src/utils/suite/dom',
-        '<rootDir>/src/utils/wallet/promiseUtils',
```
Comment (mroz22) [inline packages/suite/jest.config.js:62]: these files no longer exist

**Comment by komret** [comment]
I think including everything makes the coverage report less informative as we do not have unit tests for some parts of the codebase, most notably components. If there are tests that are not currently included in coverage reports, it may be because A) `collectCoverageFrom` is too specific, targeting individual files instead of directories (e.g. `<rootDir>/src/hooks/**` is the ideal specificity IMO), or B) the tested code is in a wrong directory (e.g. some utils are located in `components`).

**Comment by mroz22** [comment]
I don't plan to spend any time on this, so please do whatever you want. Feel free to close this

**Comment by komret** [comment]
How about this? There are a few tests in the components folder which won't be included, but that shouldn't matter for a coverage report because it should give an overview of coverage for whole parts of the codebase, not just individual files IMO 05916c5593ad9facd1aa017291bf437e7fcb8383

**Comment by mroz22** [comment]
ok

**Review by komret — APPROVED** [review]

**Pull request merged**